src/Termonad/App.hs

```diff
@@ -433,17 +439,29 @@ setupTermonad tmConfig app win builder = do
   actionMapAddAction app newTabAction
   applicationSetAccelsForAction app "app.newtab" ["<Shift><Ctrl>T"]
 
+  nextPaneAction <- simpleActionNew "nextpane" Nothing
+  void $ onSimpleActionActivate nextPaneAction $ \_ ->
+    termTogglePane mvarTMState
+  actionMapAddAction app nextPaneAction
+  applicationSetAccelsForAction app "app.nextpane" ["<Ctrl>Page_Down"]
+
+  prevPaneAction <- simpleActionNew "prevpane" Nothing
+  void $ onSimpleActionActivate prevPaneAction $ \_ ->
+    termTogglePane mvarTMState
+  actionMapAddAction app prevPaneAction
+  applicationSetAccelsForAction app "app.prevpane" ["<Ctrl>Page_Up"]
+
   nextPageAction <- simpleActionNew "nextpage" Nothing
   void $ onSimpleActionActivate nextPageAction $ \_ ->
     termNextPage mvarTMState
   actionMapAddAction app nextPageAction
-  applicationSetAccelsForAction app "app.nextpage" ["<Ctrl>Page_Down"]
+  applicationSetAccelsForAction app "app.nextpage" ["<Ctrl><Shift>Page_Down"]
```

Comment:
I agree with your reasoning here, although I don't personally use the `<Ctrl>Page_Down` key.

I imagine I might have to finally start working on https://github.com/cdepillabout/termonad/issues/83 !

Reply:
Hmm, now that I think about it, this key combination only makes sense because with the two-panes approach, there is an obvious next-pane and prev-pane. Once arbitrary splittings are allowed, we'll need key combinations to move up, down, left and right, not prev and next. So I think `<Ctrl>Page_Down` can continue to be reserved for tabs.

Reply:
I left some example solutions [here](https://github.com/cdepillabout/termonad/issues/205#issuecomment-964828382) earlier.

`tmux` by way of example, employs a "previous / next" concept so that you can use:
* `ctl+b` then `;` to toggle previous pane (effectively switches between two panes)
* `ctl+b` then `{` or `ctl+b` then `}` for moving panes
* `ctl+b` then `arrow key` for simply changing to the next pane in that direction.